lms/static/scripts/frontend_apps/utils/google-api-client.js

```diff
@@ -12,8 +12,8 @@ async function loadGAPI() {
     gapiScript.onload = () => {
       resolve(window.gapi);
     };
-    gapiScript.onerror = ({ error }) => {
```

Comment:
This is a mistake caught by TS. The `onerror` handler gets a [generic `Event`](https://w3c.github.io/uievents/#event-type-error) with no `error` property.